pix-editor/app/templates/authenticated/competence/prototypes/single/alternatives.hbs

```diff
@@ -11,10 +11,17 @@
     </div>
     {{#unless this.rightMaximized}}
       <div class="ui attached segment competence {{this.size}}">
-        <List::Alternatives @list={{this.challenge.alternatives}} />
+        <List::Alternatives @list={{this.alternatives}} />
       </div>
       <div class="ui borderless bottom attached labelled icon menu">
-        <div class="item competence-info"><div>En production : <span class="production">{{this.challenge.productionAlternatives.length}}</span></div><div>Brouillons : <span class="workbench">{{this.challenge.draftAlternatives.length}}</span></div></div>
+        <div class="item competence-info">
+          <div>
+            {{input id="hide-perime" type="checkbox" checked=this.arePerimeDeclisDisplayed}}
```

Comment:
**issue:** Le [input helper](https://api.emberjs.com/ember/5.9/classes/Ember.Templates.helpers/methods/input?anchor=input) semble être un reliquat des premières version d'Ember, à bannir bien que pas déprécier.

Ça peut être remplacé par le composant built-in Input de Ember :

```suggestion
            <Input @id="hide-perime" @type="checkbox" @checked={{this.arePerimeDeclisDisplayed}} />
```

Ça pourrait aussi être remplacer par la balise input native, la différence entre les 2 est bien expliquée sur [cette ADR PixUI](https://github.com/1024pix/pix-ui/blob/dev/docs/adr/0001-utiliser-les-balises-html-natives.md).